### DISCUSSION TITLE
How to start plotting after reboot?

### DISCUSSION THREAD

**Discussion by Purple-Tentacle** [ericaltendorf/plotman#372] [Q&A]
I want a machine start plotting after a reboot.
Of course with archiving.

Is it possible?

**Comment by altendky** [comment]
That's what system service managers are for.  You can do it presently kind of and I do intend to make plotman much more oriented in this direction.  There are a number of little pieces but we'll get there.

**Reply by BasilHorowt** [reply]
To add to that, there's an issue to track improvements to this area here: https://github.com/ericaltendorf/plotman/issues/285
@Purple-Tentacle feel free to subscribe to that issue to get updates :)

**Reply by Purple-Tentacle** [reply]
Could you eloborate on how to do it currently?

**Reply by altendky** [reply]
`plotman plot` plots and `plotman archive` archives.  for monitoring, when you run `plotman interactive` you can disable plot and archive starting by pressing `p` and `a`.  there's a pr in the works to make command line options so you can launch `interactive` with them already disabled.

on the system service side, depends what system you are on and what service manager you are using.  i don't have a lot of experience on that side.

**Reply by Purple-Tentacle** [reply]
Debian 10.
A cronjob with @reboot starring a script?
And this should include plotman plot/archive?
